Bottom: f2f91607448bedec3ff620589cab1c3502506fe6
Top:    a52b1ec21db323d90faaeb7b4565ef3a0ae66c81
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-27 12:54:44 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index b7fc04c..5a1f088 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -324,13 +324,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
